Source/ParserProperty.swift

```diff
@@ -80,43 +70,37 @@ public struct ParserProperty {
 
 // MARK: -
 
-/**
-    The parser property maker is used to define the list of properties to be validated and extracted from a object.
-    If a property is not defined in the list it will be ignored.
-*/
+/// Defines the list of properties to be validated and extracted from a object. If a property is not defined in the
+/// schema but is present in the JSON data, it will be ignored.
 public class Schema {
     var properties = [ParserProperty]()
 
-    /**
-        Creates, adds and returns a property for the specified key path, type and optionality.
-
-        NOTE: Compound key paths may be used (e.g. `address.city`). Each property name in the key path MUST be
-        separated by a `.` character.
-
-        - parameter keyPath:  Key path for property.
-        - parameter type:     Swift object type to be validated and extracted.
-        - parameter optional: Specifies if the keyPath is optional. `false` by default.
-
-        - returns: The created parser property.
-    */
+    /// Creates, adds and returns a property for the specified key path, type and optionality.
+    ///
+    /// NOTE: Compound key paths may be used (e.g. `address.city`). Each property name in the key path MUST be
+    /// separated by a `.` character.
+    ///
+    /// - parameter keyPath:  Key path for property.
+    /// - parameter type:     Swift object type to be validated and extracted.
+    /// - parameter optional: Specifies if the keyPath is optional. `false` by default.
+    ///
+    /// - returns: The created parser property.
     @discardableResult
     public func addProperty(keyPath: String, type: ParserPropertyType, optional: Bool = false) -> ParserProperty {
```

Comment:
`forKeyPath` or `keyPath`? Only ask b/c we're about to the point we're locked. 🤔